bpfd-api/src/util.rs

```diff
@@ -25,23 +33,15 @@ pub mod directories {
     pub const RTDIR_FS:         &str = "/run/bpfd/fs";
     pub const RTDIR_FS_MAPS:    &str = "/run/bpfd/fs/maps";
     pub const RTDIR_PROGRAMS:   &str = "/run/bpfd/programs";
+
     //
     // CacheDirectory: /var/cache/
+
     //
     // StateDirectory: /var/lib/bpfd/
     pub const STDIR:        &str = "/var/lib/bpfd";
     pub const STDIR_SOCKET: &str = "/var/lib/bpfd/sock";
-    //
-    // LogsDirectory:  /var/log/
 
-    // The following directories are used by bpfctl. They should be created by bpfctl service
-    // via the bpfctl.service settings. They will be manually created in the case where bpfctl
-    // is not run as a service.
     //
-    // ConfigurationDirectory: /etc/bpfctl/
-    pub const CFGDIR_BPFCTL:            &str = "/etc/bpfctl";
-    pub const CFGDIR_BPFCTL_CERTS:      &str = "/etc/bpfctl/certs/bpfctl";
-    pub const CFGPATH_BPFCTL_CERTS_PEM: &str = "/etc/bpfctl/certs/bpfctl/bpfctl.pem";
-    pub const CFGPATH_BPFCTL_CERTS_KEY: &str = "/etc/bpfctl/certs/bpfctl/bpfctl.key";
-    pub const CFGPATH_BPFCTL_CONFIG:    &str = "/etc/bpfctl/bpfctl.toml";
+    // LogsDirectory:  /var/log/
```

Comment:
Does anything actually live in /var/log/ anymore?

Reply:
No, do we want bpfd logs in something like /var/log/bpfd/?

Reply:
Removed comments for directories we aren't using.

Reply:
Other way around 😆 
Systemd service - logs go to journal
Not systemd service - logs go to stdout (or log file)